### ISSUE TITLE
Rollup options missing a dot ?

### ISSUE THREAD

**Issue by FluffyDiscord** [lhapaipai/vite-plugin-symfony#1]
Hi,
for some reason after composer update and yarn upgrade I had to change vite config build.rollupOptions.input.app to "../assets/app.ts". Without the extra dot "yarn dev" would hang forever and "yarn build" would point to invalid path

![image](https://user-images.githubusercontent.com/10332247/145270826-795fe9cc-e7d3-41f1-8cac-434c50d81bbd.png)

yarn build error
![image](https://user-images.githubusercontent.com/10332247/145271001-82c40932-0387-49c2-b8a4-2eaa5f50b72f.png)

**Comment by lhapaipai** [comment]
same issue 
with vite <= 2.6.x we need one dot
with vite 2.7.x we need two dot.

I will read the doc to understand :flushed:

**Comment by FluffyDiscord** [comment]
As a side note - you should be using exact dependency versions. Please use specific vite version as a dependency once you update this package.

**Comment by lhapaipai** [comment]
ok i will do that.

finally with v2.7.x we need to specify the path relatively to the `root` option.
```js
export default defineConfig({

  root: "./assets",
  build: {
    rollupOptions: {
      input: {
        "app": "./app.ts",
       },
    },
  }
});
```

**Comment by lhapaipai** [comment]
I'm banging my head against the walls with the version numbers,
do you think i should write this in the `package.json` ?

```json
{
  "name": "vite-plugin-symfony",
  "devDependencies": {
    "vite": "~2.6"
  },
  "peerDependencies": {
    "vite": "^2 < 2.7"
  }
}
```

I don't think it's a dependency because we can't have multiple versions of vite in our project.

**Comment by FluffyDiscord** [comment]
Why not

```json
{
"peerDependencies": {
    "vite": "2.7.*"
  }
}
```

**Comment by lhapaipai** [comment]
thank you for your advice.
I released a last version 0.1.x with vite 2.6.x in peerDependency and a version 0.2.0 with vite 2.7.x in peerDependency.
I don't know if version 0.1.6 was needed but the intention was there...
